src/components/StatusIndicator/__tests__/__snapshots__/StatusIndicator.spec.js.snap

```diff
@@ -1,14 +1,16 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`StatusIndicator renders 1`] = `
-<Fragment>
+<div
+  className="security--status-indicator"
+>
   <h1
     className="security--status-indicator__title"
   >
     Status indicator title
   </h1>
   <ul
-    className="security--status-indicator"
```

Comment:
Is there a way to avoid these class name changes? 🤔 More of a situation of "just in case" -- this component has been around for a while so I worry that a consumer may be relying on these class names for one reason or another, so this change may break a test, style override, etc.

Reply:
Good point - I'll do a cursory search for impact and make changes where necessary